builder/main.py

```
@@ -219,36 +285,115 @@ def fetch_fs_size(env):
+
+    # Parameters from platformio.ini
+    extra_args = env.GetProjectOption("custom_esp_idf_size_args", "")
+    if extra_args:
+        cmd.extend(shlex.split(extra_args))
+
+    # Command Line Parameter, after --
+    cli_args = []
+    if "--" in sys.argv:
+        dash_index = sys.argv.index("--")
+        if dash_index + 1 < len(sys.argv):
+            cli_args = sys.argv[dash_index + 1 :]
```

Ruff (0.11.9)

Local variable `cli_args` is assigned to but never used

Remove assignment to unused variable `cli_args`

(F841)